examples/ssh/ssl_certificate.js

```diff
@@ -0,0 +1,127 @@
+
+/**
+ * Domotz Custom Driver 
+ * Name: SSL Certificate Expiration check
+ * Description: This script checks the SSL certificate status for a list of target servers specified in the 'targetServerHost' and 'targetServerPort' parameters.
+ * 
+ * Communication protocol is SSH
+ * 
+ * Tested on Linux Distributions:
+ *      - Ubuntu 20.04 LTS
+ * Shell Version:
+ *      - Bash 5.1.16
+ * 
+ * Return a table containing the following columns for each server:
+ *      - Issuer
+ *      - Expiry
+ *      - Remaining days
+ *      - Is Valid
+ *      - Authorization error
+ * 
+ **/
+
+var table = D.createTable(
+    "SSL Certificates",
+    [
+        { label: "Issuer" },
+        { label: "Expiry" },
+        { label: "Remaining days", unit: "day" },
+        { label: "Is valid" },
+        { label: "Authorization error" }
+    ]
+);
+
+var serversToCheck = D.getParameter('serversToCheck');
+
+var sshConfig = {
+    username: D.device.username(),
+    password: D.device.password(),
+    timeout: 30000
```

Comment:
Its 30 seconds per server check, is that not a bit too high?

Maybe reduce the individual server check timeout to something less (eg 5 or 10 seconds depending on how fast the openssl command is) and increase the total to 2 min (see below comment)